File: adapters/qwen7b/Modelfile.go
# axon-go -- Qwen2.5-Coder com o adapter treinado nas licoes de Go do axon-lang.
# O FROM precisa ser o mesmo modelo em que o adapter foi treinado (BASE = "qwen").
FROM qwen2.5-coder:7b-instruct
ADAPTER ./go-lora.gguf

# Baixo o suficiente pra ele seguir o material recuperado em vez de inventar API.
PARAMETER temperature 0.3

SYSTEM """Voce responde em portugues do Brasil sobre programacao em Go. Use o material
fornecido no prompt. Se ele nao cobrir a pergunta, diga isso em vez de inventar."""

File: adapters/qwen7b/Modelfile.php
# axon-php -- Qwen2.5-Coder 7B com o adapter treinado nas licoes de PHP.
# O FROM tem que ser o modelo em que o adapter foi treinado: adapter nao migra de base.
FROM qwen2.5-coder:7b-instruct
ADAPTER ./php-lora.gguf

# Baixo para ele seguir o material recuperado em vez de completar de memoria.
PARAMETER temperature 0.3

SYSTEM """Voce responde em portugues do Brasil sobre PHP, de forma direta e
tecnica. Use o material fornecido no prompt. Se ele nao cobrir a pergunta, diga isso em
vez de inventar."""
